### DISCUSSION TITLE
Stackblitz example of "Navbar with menu and submenu" does not seem to work

### DISCUSSION THREAD

**Discussion by Norfeldt** [saadeghi/daisyui#2357] [Q&A] [answered]
Just learning DaisyUI and trying out some of the components on stackblitz. Tried the [navbar with menu and submenu](https://dev.daisyui.com/components/navbar/#navbar-with-menu-and-submenu) and noticed that clicking on the "Parent" looks different on stackblitz than on the component demo site?

https://stackblitz.com/edit/daisyui-vue-vite-ryhzix?file=src%2FApp.vue

Am I missing something?

**Comment by saadeghi** [comment, marked as answer]
That stackblitz link is using version `2.6.0` of daisyUI. installing the latest version (`npm i -D daisyui@latest`) would fix the issue.

**Reply by Norfeldt** [reply]
thank you 🙏
